### ISSUE TITLE
Don't crash on duplicated filetypes

### ISSUE THREAD

**Issue by bmihaila-synopsys** [spdx/tools-python#630]
Some SBOM creation tools create `fileTypes` entries which might contain duplicates, e.g.
```
"fileTypes": [
                "SOURCE",
                "TEXT",
                "SOURCE",
            ],
```

The MR https://github.com/spdx/tools-python/pull/310/ added code for supporting multiple fileTypes but the code is too strict in that it does not allow duplicates. It throws a `raise CardinalityError("File::FileType")` in case of dupes. The spec does not forbid this https://spdx.github.io/spdx-spec/v2.2.2/file-information/#83-file-type-field so the parser should not be that strict and crash on such files. I see a similar behavior for `licenseInfoInFiles` which do not raise an exception on duplicates.

**Comment by meretp** [comment]
Thanks for opening this issue. In the meanwhile we merged a refactored version of the codebase to `main` but I assume that you are refering to the current release v0.7.1?

**Comment by bmihaila-synopsys** [comment]
btw, the link in the README towards the migration is broken https://github.com/spdx/tools-python/wiki/How-to-migrate-from-0.7-to-1.0

**Comment by meretp** [comment]
> Thanks. Will try the alpha, was not aware of that. Regarding the fix. My oneliner fix was to just not do that check and raise the Cardinality error, i.e. just remove that code. Depending on how much you want to "fix" somewhat broken SBOMs you could just use a set instead of a list and hence remove duplicates but as you mention, that old code branch is bugifx only so I guess just skipping the raise is the simpler fix.

Yes, I agree on that, simply skip should be sufficient.

> btw, the link in the README towards the migration is broken https://github.com/spdx/tools-python/wiki/How-to-migrate-from-0.7-to-1.0

Yeah I also noticed that and opened a PR #631 for that.

**Comment by armintaenzertng** [comment]
fixed by #671
